Addons.ini

```diff
@@ -77,7 +77,7 @@ RepositoryUrl=https://github.com/4lex4nder/ReshadeEffectShaderToggler
 [11]
 PackageName=IGCS Connector by Otis_Inf
 PackageDescription=Connects ReShade with a photomode mod built with IGCS to exchange data and issue actions
-DownloadUrl64=https://github.com/FransBouma/IgcsConnector/releases/download/v251/IGCSConnector_v251.zip
+DownloadUrl64=https://github.com/FransBouma/IgcsConnector/releases/download/v251/IGCSConnector_v252.zip
```

Comment:
That should be https://github.com/FransBouma/IgcsConnector/releases/download/v252/IGCSConnector_v252.zip I believe, this link doesn't work at least

Reply:
fixed!